### PR TITLE
Exercise3

### PR DESCRIPTION
**Pull request by SiarheiKhmalEY**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        myObjectsToScan.add(new SecondClassImpl());
+        myObjectsToScan.add(new ThirdClassImpl());
+        myObjectsToScan.add(new FourthClassImpl());
+
+        List<Class<? extends Annotation>> customAnnotationList = new ArrayList<>();
+        customAnnotationList.add(Private.class);
+        customAnnotationList.add(Public.class);
+
+//        Task iii.1
+//        getAllClassesWithAnnotation(ScanMe.class, myObjectsToScan);
```
Comment (jaroslawwoloch) [inline src/Main.java]: why commented?
remove commented code

File: src/Main.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+//        Task iii.3
+//        getAllMethodsWithoutCustomAnnotationInClassWithoutAnnotation(customAnnotationList, myObjectsToScan);
+
+//        Task iii.4
+//        getAllScannedClasses(myObjectsToScan);
+
+    }
+
+    public static void getAllClassesWithAnnotation(Class<? extends Annotation> classAnnotation, List<MyInterface> elementList) {
+        for (MyInterface element : elementList
```
Comment (jaroslawwoloch) [inline src/Main.java]: formating - form method braces in same line but on for loop in next line - correct this

File: src/Main.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+//        Task iii.3
+//        getAllMethodsWithoutCustomAnnotationInClassWithoutAnnotation(customAnnotationList, myObjectsToScan);
+
+//        Task iii.4
+//        getAllScannedClasses(myObjectsToScan);
+
+    }
+
+    public static void getAllClassesWithAnnotation(Class<? extends Annotation> classAnnotation, List<MyInterface> elementList) {
```
Comment (jaroslawwoloch) [inline src/Main.java]: move to non static method

**Review by jaroslawwoloch — CHANGES_REQUESTED** [review]
remove unnecessary files from repo

File: src/Main.java (hunk 1 above)
Comment (SiarheiKhmalEY) [reply src/Main.java]: It's clarify which line run task. Anyway already deleted. Thank you

File: src/Main.java (hunk 2 above)
Comment (SiarheiKhmalEY) [reply src/Main.java]: Sorry, my bad. Already done.

File: src/Main.java (hunk 3 above)
Comment (SiarheiKhmalEY) [reply src/Main.java]: Done. Thank you.

**Comment by SiarheiKhmalEY** [comment]
> 
Already deleted. Thank you

**Review by jaroslawwoloch — APPROVED** [review]

**Pull request merged**